viziphant/gpfa.py

```diff
@@ -415,10 +423,16 @@ def plot_trajectories(returned_data,
         neo.Segment including the neo.Event with a specified
         `neo_event_name`.
         Default: None
-    neo_event_name : str or None, optional
-        A string specifying the name of the neo.Event which should be used
-        to identify the event times and labels of the `relevant_events`.
-        Default: None
+    neo_event_dict : dict, optional
+        Dictionary of names and properties of the events that should be plotted 
+        onto each trajectory.
+        The key specifies the label appearing in the legend. The item specifies
+        the neo_event_properties, which are used by neo.utils.get_events()
+        to filter the event. 
+        In case more than one event per dict-entry is available, only the first
+        one is chosen to be plotted. Please specify more properties to narrow
+        down the choice in that case.
+        Default: {}
     relevant_events : list of str or None, optional
```

Comment:
remove `relevant_events` parameter from the docstring, since it has been removed and replaced by `neo_event_dict`?